build.gradle

```diff
@@ -262,8 +262,8 @@ dependencies {
     compile 'org.jooq:jool-java-8:0.9.14'
     compile 'com.github.hmcts:ccd-case-document-am-client:1.7.1'
 
-    testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.2.0'
-    testCompile group: 'com.github.hmcts', name: 'befta-fw', version: '8.2.0'
+    testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.7.4-prerelease'
```

Comment:
This needs to be updated to a full release version of the hmcts/ccd-test-defintions before going into master. see hmcts/ccd-test-definitions#33.

Reply:
Resolved in 6505766b9fccdc8bfa6fa7cb55cf1a9ceadf86b4